views/js/test/qtiCreator/renderers/static/tooltip/test.js

```diff
@@ -38,31 +34,40 @@ define([
     QUnit.module('Creator Renderer');
 
     QUnit.test('Display and play', function(assert) {
-        var ready = assert.async();
-        var $outsideContainer = $('#outside-container'),
-            config = {
-                qtiCreatorContext: qtiCreatorContextMockFactory()
+        const ready = assert.async();
+        const config = {
+            qtiCreatorContext: qtiCreatorContextMockFactory()
+        };
+
+        const areaBroker = {
+            getElementPropertyPanelArea() {
+                return $('#qunit-fixture');
             },
-            areaBroker = areaBrokerFactory({
-                $brokerContainer: $outsideContainer,
-                mapping: {
-                    itemPanel: $('.item-editor-item'),
-                    contentCreatorPanel: $('#item-editor-panel'),
-                    toolbar: $('#toolbar-top')
-                }
-            }),
-            tooltipContent = 'my tooltip <strong>content</strong>',
-            tooltipSerial = '_tooltip_4568613547893',
-            tooltipId = 'tooltip_123456',
+            getContentCreatorPanelArea() {
+                return $('#item-editor-panel');
+            },
+            getItemPanelArea() {
```

Comment:
It returns with the same element like `getContentCreatorPanelArea`, only the selector is different. Is that correct?

Reply:
yes, I've just took the selectors from the original test